google-cloud-error_reporting/lib/google/cloud/error_reporting/middleware.rb

```diff
@@ -73,7 +74,7 @@ def initialize app, error_reporting: nil, project_id: nil,
                         ENV["GOOGLE_CLOUD_PROJECT"] ||
                         Google::Cloud::Core::Environment.project_id
 
-          raise ArgumentError, "project_id is required" if @project_id.nil?
+          fail ArgumentError, "project_id is required" if @project_id.nil?
```

Comment:
Perhaps https://github.com/bbatsov/rubocop/blob/ec3123fc3454b080e1100e35480c6466d1240fff/lib/rubocop/cop/style/signal_exception.rb#L10

https://github.com/bbatsov/rubocop/blob/ec3123fc3454b080e1100e35480c6466d1240fff/lib/rubocop/cop/style/signal_exception.rb#L11

explain it best. The README seems to be out-of-date.